libs/src/services/ethWeb3Manager/index.js

```diff
@@ -58,12 +59,12 @@ class EthWeb3Manager {
     const gasLimit = txGasLimit || await estimateGas({
       method: contractMethod,
       from: this.ownerWallet,
-      gasLimitMaximum: HIGH_GAS_PRICE
```

Comment:
this was a bug that allowed gas limit to be set to 39 billion which is more than the 30 million block gas limit